scripts/config.py

```diff
@@ -159,46 +159,58 @@ def realfull_adapter(_name, active, section):
         return active
     return True
 
+# The goal of the full configuration is to have everything that can be tested
+# together. This includes deprecated or insecure options. It excludes:
+# * Options that require additional build dependencies or unusual hardware.
+# * Options that make testing less effective.
+# * Options that are incompatible with other options.
+# * Options that remove features.
+# * Options that are variants, so that we need to test both with and without.
```

Comment:
According to our discussion, we should probably update this line, because "we need to test both with and without" applies to many things (ideally everything if we could). Perhaps "Options that are variants, in that either value will exclude some code (or otherwise limit testing)."